cmd/stakercli/daemon/daemoncommands.go

```diff
@@ -133,6 +139,37 @@ var stakeCmd = cli.Command{
 	Action: stake,
 }
 
+var stakeFromPhase1Cmd = cli.Command{
+	Name:      "stake-from-phase1",
+	ShortName: "stfp1",
+	Usage: "\nstakercli daemon stake-from-phase1 [fullpath/to/global_parameters.json]" +
+		" --staking-transaction-hash [txHashHex] --staker-address [btcStakerAddrHex] --tx-inclusion-height [blockHeightTxInclusion]",
+	Description: "Creates a Babylon BTC delegation transaction from the Phase1 BTC staking transaction",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  stakingDaemonAddressFlag,
+			Usage: "full address of the staker daemon in format tcp:://<host>:<port>",
+			Value: defaultStakingDaemonAddress,
+		},
+		cli.StringFlag{
+			Name:     stakingTransactionHashFlag,
+			Usage:    "Hash of original staking transaction in bitcoin hex format",
+			Required: true,
+		},
+		cli.StringFlag{
+			Name:     stakerAddressFlag,
+			Usage:    "BTC address of the staker in hex",
+			Required: true,
+		},
+		cli.Uint64Flag{
+			Name:     txInclusionHeightFlag,
```

Comment:
I wodner whether this param is necessary i.e we have staking tx hash, we can retrieve staking transaction from the btc chain and check its height and based on that chose the param.

Non-blocking comment it can be tackled as future improvement imo.